openedx/openedx-platform
# Taniwha/bb 2597 [TNL-7061] Remove the old course info page post launch (take 2)

Once the merged home/course tab is launched, remove all the code for the old course info page.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-2597

**Dependencies**: None

**Screenshots**: Before (left) and after (rignt) of the user-visible changes to the UI
![course_info](https://user-images.githubusercontent.com/1664867/87010480-93e16700-c201-11ea-9321-9d04c76ee417.png)


**Sandbox URL**: TBD?

**Merge deadline**: "None"

**Testing instructions**:

1. In studio, go to a course page
2. In the Content menu, the Updates item should be gone
3. Attempting to go to the old course info page should result in a 404 (example url http://localhost:18010/course_info/course-v1:blank+1234+2020)

**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
